llvm/lib/Analysis/BasicAliasAnalysis.cpp

```diff
@@ -1159,8 +1163,6 @@ AliasResult BasicAAResult::aliasGEP(
     APInt &Off = DecompGEP1.Offset;
 
     // Initialize for Off >= 0 (V2 <= GEP1) case.
-    const Value *LeftPtr = V2;
-    const Value *RightPtr = GEP1;
```

Comment:
Not related to this change -- I've landed this separately in https://github.com/llvm/llvm-project/commit/228753290b7426bf3eda7c4397aa74f0c67d07d7.

Reply:
Nice, thank you - rebased on this.